draft-ietf-quic-recovery.md

```diff
@@ -371,16 +371,23 @@ retain sufficient history is an open research question.
 
 ## Estimating min_rtt {#min-rtt}
 
-min_rtt is the minimum RTT observed over the lifetime of the connection.
-min_rtt is set to the latest_rtt on the first sample in a connection, and to the
-lesser of min_rtt and latest_rtt on subsequent samples.
+min_rtt is the minimum RTT observed for a given congestion control context.
```

Comment:
min_rtt is only used for loss recovery in this doc. I think it's more appropriate to say RTT estimator context in general.

Reply:
When you say "congestion control context", why not "path"?

Reply:
Good question, I'll talk with Eric and Jana in person about this one and we can figure out some consistent terminology.